packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -159,7 +159,7 @@ class _RenderCheckbox extends RenderToggleable {
     double rectSize = _kEdgeSize - inset * _kStrokeWidth;
     Rect rect = new Rect.fromLTWH(offsetX + inset, offsetY + inset, rectSize, rectSize);
 
-    RRect outer = new RRect.fromRectXY(rect, _kEdgeRadius, _kEdgeRadius);
+    RRect outer = new RRect.fromRectAndRadius(rect, const Radius.circular(_kEdgeRadius));
```

Comment:
`kTrackRadius` is used in a calculation below. Maybe we should keep it a `double`?